libs/components/help-inline/src/lib/modules/help-inline/help-inline.component.html

```diff
@@ -2,7 +2,11 @@
   class="sky-help-inline"
   type="button"
   [attr.aria-controls]="
-    ariaControls | skyHelpInlineAriaControls: popoverId : helpKey
+    ariaControls
```

Comment:
Should `aria-haspopup` be undefined if the `widgetReadyStateChange` hasn't emitted `true`? If there is no given widget - then there wouldn't be a popup that is ready, correct?

Reply:
`aria-haspopup` and `aria-controls` are related concepts, but they can be implemented separately and still be considered accessible. For example, we might have a button that generates a menu after being clicked. We'd set `aria-haspopup` on the button to communicate to the user that the button will open a listbox, but we wouldn't set `aria-controls` since the menu element does not yet exist in the DOM.